src/components/chart/_macro-options.md

```diff
@@ -1,25 +1,24 @@
-| title                   | string  | true     | The main title of the chart.                                                                                                                                                                 |
-| subtitle                | string  | true     | A subtitle that appears under the main title.                                                                                                                                                |
-| id                      | string  | true     | A unique identifier for the chart instance.                                                                                                                                                  |
-| caption                 | string  | false    | A caption providing additional context for the chart.                                                                                                                                        |
-| description             | string  | false    | A textual description of the chart for screen readers.                                                                                                                                       |
-| download                | object  | false    | Object for (download)[#download] options.                                                                                                                                                    |
-| legend                  | boolean | false    | Whether the legend is displayed. Defaults to `true`.                                                                                                                                         |
-| yAxis                   | object  | true     | Defines the vertical axis [y-axis](#y_axis) configuration parameters                                                                                                                         |
-| xAxis                   | object  | true     | Defines the horizontal axis [x-axis](#x_axis) configuration parameters                                                                                                                       |
-| series                  | array   | true     | The data [series](#series) to be plotted, including labels and values.                                                                                                                       |
-| useStackedLayout        | boolean | false    | Determines whether the chart should use a stacked layout. It is useful only for bar and column charts.                                                                                       |
-| annotations             | array   | false    | An array of annotations.                                                                                                                                                                     |
```

Comment:
This update removes a duplicate entry for `annotations` which probably happened in a merge at some point.